resources/assets/coffee/_classes/forum-cover.coffee

```diff
@@ -62,24 +62,23 @@ class @ForumCover
     @uploadButton.length > 0
 
 
-  toggleModal: (e) =>
-    e.preventDefault()
+  isModalOpen: (isModalOpen) =>
+    return false if !@hasCoverEditor()
 
-    if @header[0]._open
-      @closeModal()
-    else
-      @openModal()
+    if isModalOpen?
```

Comment:
the attribute doesn't support anything but string to begin with. Also I changed it to `0` instead of `undefined`.